src/Annotation/ApiResource.php

```diff
@@ -112,67 +109,81 @@ final class ApiResource
         'paginationEnabled',
         'paginationFetchJoinCollection',
         'paginationItemsPerPage',
-        'maximumItemsPerPage',
         'paginationMaximumItemsPerPage',
         'paginationPartial',
         'paginationViaCursor',
         'routePrefix',
+        'security',
+        'securityMessage',
+        'securityPostDenormalize',
+        'securityPostDenormalizeMessage',
         'sunset',
         'swaggerContext',
         'validationGroups',
     ];
 
     /**
-     * @var string
+     * @see https://api-platform.com/docs/core/operations/#operations
```

Comment:
Can we remove all the pointless hashes that are just top of the page anyway? :smile:

i.e. it should be https://api-platform.com/docs/core/operations/